charts/esphome/item.yaml

```diff
@@ -0,0 +1,3 @@
+categories:
+  - tools
```

Comment:
I think we need to document catagories and what they represent someday in the wiki.
For now this is fine.